core/services/request/Response.php

```diff
@@ -136,4 +137,19 @@ public function deactivatePlugin()
     {
         $this->deactivate_plugin = true;
     }
+
+    /**
+     * Registers EE valid style attributes
+     */
+    public function registerValidStyles()
+    {
+        add_filter(
+            'safe_style_css',
+            function ($styles) {
+                $styles[] = 'display';
+                $styles[] = 'visibility';
+                return $styles;
+            }
+        );
+    }
 }
```

Comment:
@tn3rb I'm pretty sure that you might know a better place to register this filter. I will change its location after receiving your suggestion.